packages/language-service/lib/commands.js

```diff
@@ -146,3 +139,12 @@ export function createSyntaxToggle(context, type, separator) {
     }
   }
 }
+
+export const implementations = {
+  'mdx.toggleDelete': createSyntaxToggle('delete', '~'),
+  'mdx.toggleEmphasis': createSyntaxToggle('emphasis', '_'),
```

Comment:
`*`

Reply:
It doesn’t really matter. I know remark stringifies uses `*` in its output by default. I personally feel `*` is a bit overloaded. Emphasis and strong text share the same characters, but `*` is also used for list items. Also Prettier uses `_`, which I think added to making that character option more popular.

TL;DR I prefer `_`, but we can change it to `*` if you feel very strongly about it. This should be done in a follow-up PR though, as it’s an unrelated change.

Reply:
Why does it not matter? Yes, it matters. `_` and `*` get parsed differently.

Reply:
Both exist, but have relatively rare use cases in natural language. It’s really just a preference, and it’s ok your preference differs from mine.

I will change it in a follow-up PR anyway, as I believe the strongest argument is that it’s good to match the `remark-stringify` default. If we enable configurable formatting for #456, I think we should respect the same configuration optionn for syntax toggles.